how-to/privileged.md

```diff
@@ -0,0 +1,45 @@
+# Privileged Kata Containers
+
+Kata Containers supports creation of containers that are "privileged" - i.e. have additional capabilities and access
+that isn't normally granted.
+
+* [Caveats](#caveats)
+    * [Host Devices](#host-devices)
+        * [Containerd and CRI](#containerd-and-cri)
+
+## Caveats
+
+Whilst this functionality is supported, it can decrease the security of Kata Containers if not configured correctly.
+
+### Host Devices
+
+By default, when privileged is enabled for a container, all of the `/dev/*` block devices from the host are mounted
+into the guest. This will allow the privileged container inside the Kata guest to gain access to mount any block device 
+from the host, a potentially undesirable side-effect that decreases the security of Kata.
+
+The following sections document how to configure this behaviour in different container runtimes.
+
+#### Containerd and CRI
+
+The Containerd CRI allows configuring the privileged host devices behaviour for each runtime in the CRI config. This is 
+done with the `privileged_without_host_devices` option. Setting this to `true` will disable hot plugging of the host 
+devices into the guest, even when privileged is enabled.
+
+See below example config:
+
+```toml
+[plugins]
+  [plugins.cri]
+   [plugins.cri.containerd]
+    [plugins.cri.containerd.default_runtime]
```

Comment:
According https://github.com/containerd/cri/blob/master/docs/config.md, the  'default_runtime' would be deprecated soon, so could you use `default_runtime_name` and `plugins.cri.containerd.runtimes` instead?

Reply:
Yes, indeed. `default_runtime` will be gone so better use `plugins.cri.containerd.runtimes.runc` instead.